### DISCUSSION TITLE
Would printing progress messages to stderr instead of stdout be a breaking change?

### DISCUSSION THREAD

**Discussion by rohitpaulk** [gleam-lang/gleam#3496] [General]
When running `gleam run`, it's difficult to pipe output since compiler progress messages are printed  to stdout. I'm working on https://github.com/gleam-lang/gleam/issues/2299, but just realized that this wouldn't be so much of a problem if the messages were printed to stderr instead of stdout. 

Since Gleam is post-v1, wanted to ask about this first - would changing these messages to print to stderr instead of stdout be considered a breaking change? 

https://github.com/gleam-lang/gleam/blob/8743158ef7ac8c8bebd045fadb4f4775a0e7cc8d/compiler-cli/src/cli.rs#L122-L124

**Comment by lpil** [comment]
Sounds like a good idea to me

https://github.com/gleam-lang/gleam/issues/3497
